src/js/components/Distribution/index.d.ts

```diff
@@ -8,7 +8,7 @@ export interface DistributionProps {
   children?: ((...args: any[]) => any);
   fill?: boolean;
   gap?: "xsmall" | "small" | "medium" | "large" | "xlarge" | string;
-  values: {value: number}[];
+  values: {value: number, color: string}[];
```

Comment:
Thanks for catching the missing color!! 💯 
grommet `color` may also be an object that includes a color mapping for dark and light colors, so I added a commit to cover an additional case, please approve it or add to it.
To see an example of the color object usage, check out this built-in example https://v2.grommet.io/distribution?c=DwEQlgzgLgTmBGBXKYD2A7AUAAmwNwEMAbRAUwgF4BvAbR12yv2LIC5sBWABgBpsBjVEVQx2VImADmACyjsA5AX79S6KAFoATPL4ATAjADWCiTI0BmeQF9sVnvVxNCJUu3Ndb9ho2Yv2m3gEhEQUlFTV1AEZ5TwcfZzZsSMDBYVFseVNZdUtY7ycWV04+VJCM+BgCdF0Yu3oAXStMAD56KgTSbApm7AAKOOAAIVQAD2wABwJdCgAiCABbYiIZ7HglQ0kYVERq6g6AOlKYGwAzMCIiVu9cYAAVUhGobAgwAC9SWaIDSVIZ5vbCvsOlZgAB6e6PK4MMHDEZQgCUTTB4GgcCQKAwzSAA